classes.py

```diff
@@ -0,0 +1,82 @@
+import datetime
+from dataclasses import dataclass
+# from mpydev import BioPac as bp
+import numpy as np
+from datetime import date
+
+
+@dataclass
+class Subject:
+    Subject_ID: int
+    Subject_ID = '0'
+    Subject_Test_Date = date.today()
+    Subject_Key = f'{Subject_ID}({Subject_Test_Date})'
+
+
+class SubjectHandle(Subject):
+    def identify_subject(self):
+        subject_id = input('Insert subject ID')
+        if isinstance(subject_id, int):
+            self.Subject_ID = subject_id
+        else:
+            raise ValueError('The Subjects ID must be an integer')
+            self.identify_subject()
+
+    def get_subject_id(self):
+        return self.Subject_ID
+
+    def get_subject(self):
+        return self.Subject_Key
+
+
+@dataclass
+class TestData(SubjectHandle):
+    Test_Data: dict
+
+
+class TestDataHandler(TestData):
+
+    def initialize_test(self):
+        self.identify_subject()
+        key = self.Subject_Key()
+        self.Test_Data = {key: {}}
+
+    def first_timestamp(self, first_timestamp_list: dict):
+        self.Test_Data[self.get_subject()]['Timestamps'] = first_timestamp_list
+
+    def timestamp_add(self, timestamp: dict):
+        self.Test_Data[self.get_subject()]['Timestamps'].update(timestamp)
+
+    def database_commit(self):
+        with open(f'{self.get_subject()}.txt') as json:
+            json.dump(self.Test_Data)
+
+
+@dataclass
+class TimeStamps(TestDataHandler):
+    timestamp: int or float or str
+    ts_dict: dict
```

Comment:
These names aren't very descriptive, but I am guessing timestamp dictionary and the numbered ones are for rounds, the purpose of these is very unclear at this stage.

Also again we shouldn't be assigning variables like this, you're better to create methods or an init to handle custom assignments.

Reply:
Ok, so these are the possible timestamps: baseline, not only for the very start of the first 24 rounds, but also for the whitespaces, as those are also baselines, the first 24 are baselines for peripherals and the whitespaces ar baselines for a specific EEG application, Q are the questions for the first 24 rounds, merely for if I choose to divide the baseline per questions later on, and the last one, IAPS is the images, I will number the 20 images one to twenty